R/utils.R

```diff
@@ -165,3 +169,106 @@ random_name <- function(prefix = "") {
   name <- paste0(sample(vals, 10, replace = TRUE), collapse = "")
   paste0(prefix, "odbc_", name)
 }
+
+# apple + spark drive config (#651) --------------------------------------------
+configure_spark <- function(call = caller_env()) {
+  if (!is_macos()) {
+    return(invisible())
+  }
+
+  unixodbc_install <- locate_install_unixodbc()
+  if (identical(unixodbc_install, character(0))) {
+    abort(
+      c(
+        "Unable to locate the unixODBC driver manager.",
+        i = "Please install unixODBC using Homebrew with `brew install unixodbc`."
+      ),
+      call = call
+    )
+  }
```

Comment:
Should this go into the generic `dbConnect()` method? It will affect all drivers, not just databricks.

Reply:
I had wondered the same thing. The only reason I didn't is because this check is both relatively invasive and ad-hoc, but I'm game to move it over if you are.